client/core/core.go

```diff
@@ -1087,6 +1087,19 @@ type Config struct {
 	TorIsolation bool
 	// Language. A BCP 47 language tag. Default is en-US.
 	Language string
+
+	// NoAutoWalletLock instructs Core to skip locking the wallet on shutdown.
+	// This can be helpful if the user wants the wallet to remain unlocked. e.g.
+	// They started with the wallet unlocked, or they intend to start Core again
+	// and wish to avoid the cost of unlocking a locked the wallet on startup.
+	NoAutoWalletLock bool // zero value is legacy behavior
```

Comment:
Several minutes in bad cases.  Despite this apparently being fixed in bitcoin core in https://github.com/bitcoin/bitcoin/commit/9abed46871a7f44277a0a59c645a8593ce9e5628, in real life large wallets it can take exorbitant amounts of time to unlock a wallet, an operation that is not necessary unless dex locks it.... which is presumptuous given it's an external self-managed wallet.